Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/Batching/README.md

```diff
@@ -0,0 +1,23 @@
+# WinML Samples Gallery: Batching
+ Perform infererence over multiple inputs at once to increase runtime performance.
+ 
+ Use this sample to compare inference runtime performace with and without batched inputs. 50 images will be passed through SqueezeNet for classification. Set a batch size and click Start Inference to compare.
+ 
```

Comment:
Can you add a TOC, and a brief explanation with more detail on why and where to look in the code to understand things?

Also links would be awesome to get people started: https://docs.microsoft.com/en-us/uwp/api/windows.ai.machinelearning.learningmodelsessionoptions.batchsizeoverride

Reply:
Done.